src/utils/winston.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line import/no-unresolved
+import { createLogger, format, transports } from 'winston';
+
+const { combine, timestamp, prettyPrint } = format;
+const logger = createLogger({
+  format: combine(
+    timestamp(),
+    prettyPrint(),
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: './error.log', level: 'error' }),
```

Comment:
Really good job here Mastel. We need a `logs` folder at the project root which would contain both files. The log files should be `.gitignore`d, once that is done we need a way to initialise the log files automatically. I suggest we write a short script to create these log files when they don't exist yet.

Reply:
Okay. Let me work on that now